doc/man3/BIO_ctrl.pod

```diff
@@ -81,6 +86,12 @@ sending. Otherwise, it returns zero.
 BIO_get_ktls_recv() returns 1 if the BIO is using the Kernel TLS data-path for
 receiving. Otherwise, it returns zero.
 
+BIO_get_conn_mode() returns the BIO connection mode. BIO_set_conn_mode() sets
+the BIO connection mode.
+
+BIO_set_tfo() disables TCP Fast Open when b<onoff> is 0, and enables TCP Fast
+Open when b<onoff> is nonzero.
```

Comment:
`B<>` or even better `I<>` but that would require changing the argument emphasis for the other functions in the file to `I<>` as well to avoid mixed style within the manpage.

Reply:
Unfortunately, argument highlighting (either via `B<>` or `I<>`) is inconsistent within the documentation, sometimes within the same file... I can at least make this `B<>`.